com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/PyFunctionHashRemover/PyFunctionHashRemover.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" ?>
+<operatorModel
```

Comment:
No, SPL standard toolkit Filter.

tupleMutationAllowed=false is unrelated, it means that downstream fiused operators are not allowed to modify the tuple reference they receive.